bin/cluster

```diff
@@ -316,7 +316,7 @@ This wrapper is overriding the following ansible variables:
         meta_parser.add_argument('cluster_id', help='prefix for cluster VM names')
 
     meta_parser.add_argument('-t', '--deployment-type',
-                             choices=['origin', 'online', 'enterprise', 'atomic-enterprise', 'openshift-enterprise'],
```

Comment:
@abutcher yes, none of the provider playbooks would succeed with deployment_type=enterprise :)